### ISSUE TITLE
Further experiments advise

### ISSUE THREAD

**Issue by parap1uie-s** [HuCaoFighting/Swin-Unet#1]
Nice work!

However, the listed experiments shown the U-shape Swin Transformer could capture global and local information. However, its ability to capture small target information has not yet been shown, i.e, further experiment on `lung nodule segmentation` task using `LIDC` dataset would be helpful.

**Comment by Siyuan89** [comment]
I agree with this comment. If you only use transformers for segmentation, it will not be able to capture the small details because transformers are not made for such a task. Convolution is needed to retain the lost local information.

**Comment by parap1uie-s** [comment]
> I agree with this comment. If you only use transformers for segmentation, it will not be able to capture the small details because transformers are not made for such a task. Convolution is needed to retain the lost local information.

Even if the outperformance is not achieved, such an experiment is valuable: Point out the limitations of the proposed method and provide the possibility for improvement.

**Comment by Acmenwangtuo** [comment]
I also use transformer to segment the nucleis ,but i can't get a better perfomance. Pure transformer suit this small objects too?

**Comment by Acmenwangtuo** [comment]
> I agree with this comment. If you only use transformers for segmentation, it will not be able to capture the small details because transformers are not made for such a task. Convolution is needed to retain the lost local information.

i meet the same question.

**Comment by swjtulinxi** [comment]
the transformer can't capture small details ?I have read a lot of segmentation papers based on it, and the results are OK
